app/main/views/feedback.py

```diff
@@ -28,7 +28,8 @@ def send_feedback():
     flash(Markup(
         """Thank you for your message. If you have more detailed feedback, please email
         <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> or
-        <a target="_blank" rel="external noopener noreferrer" href="https://airtable.com/shrkFM8L6Wfenzn5Q">take
+        <a target="_blank" rel="external noopener noreferrer"
+        href="https://www.digitalmarketplace.service.gov.uk/user/notifications/user-research">take
```

Comment:
We have an external route for this (`dmutils.external.user_research_consent`), so we should be able to avoid hardcoding the URL.

Reply:
Ahh, thank you 👍